--- bin/coffee.orig	2010-11-24 22:38:53.000000000 +0100
+++ bin/coffee	2010-11-24 22:39:38.000000000 +0100
@@ -2,6 +2,5 @@
 
 var path = require('path');
 var fs   = require('fs');
-var lib  = path.join(path.dirname(fs.realpathSync(__filename)), '../lib');
 
-require(lib + '/command').run();
+require('coffee-script/command').run();
--- bin/cake.orig	2010-11-24 22:38:48.000000000 +0100
+++ bin/cake	2010-11-24 22:39:19.000000000 +0100
@@ -2,6 +2,5 @@
 
 var path = require('path');
 var fs   = require('fs');
-var lib  = path.join(path.dirname(fs.realpathSync(__filename)), '../lib');
 
-require(lib + '/cake').run();
+require('coffee-script/cake').run();
